src/sage/topology/simplicial_complex.py

```diff
@@ -4075,51 +4080,51 @@ def fundamental_group(self, base_point=None, simplify=True):
         EXAMPLES::
 
             sage: S1 = simplicial_complexes.Sphere(1)
-            sage: S1.fundamental_group()
+            sage: S1.fundamental_group()                                                # optional - sage.graphs sage.groups
             Finitely presented group < e |  >
 
         If we pass the argument ``simplify=False``, we get generators and
         relations in a form which is not usually very helpful. Here is the
         cyclic group of order 2, for instance::
 
-            sage: RP2 = simplicial_complexes.RealProjectiveSpace(2)
-            sage: C2 = RP2.fundamental_group(simplify=False)
-            sage: C2
-            Finitely presented group < e0, e1, e2, e3, e4, e5, e6, e7, e8, e9 | e0, e3, e4, e7, e9, e5*e2^-1*e0, e7*e2^-1*e1, e8*e3^-1*e1, e8*e6^-1*e4, e9*e6^-1*e5 >
-            sage: C2.simplified()
+            sage: RP2 = simplicial_complexes.RealProjectiveSpace(2)                     # optional - sage.libs.pari
+            sage: C2 = RP2.fundamental_group(simplify=False); C2                        # optional - sage.graphs sage.groups sage.libs.pari
+            Finitely presented group < e0, e1, e2, e3, e4, e5, e6, e7, e8, e9 | e0, e3,
```

Comment:
Does L4090 need `sage.libs.pari`? Or is it L4091?

Reply:
Looks like a mistake. I have removed `sage.libs.pari` from all lines in this block